test/bugs/apps_1052_optional_compound_input_wdl10_input.json

```diff
@@ -0,0 +1,3 @@
+{
+  "apps_1052_optional_compound_input_wdl10.t1" : {"inner": {}}
```

Comment:
Would that be better if we test it with a valid inner? like t1.inner.bb=111. In that case we are expecting the result a to be 111 instead of 2 and we can check both cases where optional is null and defined. but that is something minor :)

Reply:
This is a good point. I've run such a test manually and got the expected result
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/2283292/155821189-2f475106-e9be-4e64-b79e-6d3bae461e15.png">
I'll add an automatic test separately.

Reply:
The integration tests passed in the previous commit so I'll go ahead and merge.